WordPress/gutenberg
# File Block: Add a deprecation for the button element class name

## What?
Adds a deprecation for the file block button element class name. I should have done this in https://github.com/WordPress/gutenberg/pull/41239.

## Why?
When we change the save function of a block we need to add a deprecation so that the new code knows how to interpret the old markup.

## How?
This just adds a copy of the save function and attributes of the block in the state they were in before https://github.com/WordPress/gutenberg/pull/41239.

## Testing Instructions
- Using WordPress 6.0, add a file block to a post
- Activate Gutenberg
- Reload the post editor
- Confirm that you see a block validation error:
<img width="853" alt="Screenshot 2022-10-20 at 13 34 55" src="https://user-images.githubusercontent.com/275961/196949913-2481ec2f-c36a-4cc2-aea2-c25e6d0bd5fc.png">
- Checkout this branch
- Confirm that the error is gone: 
<img width="475" alt="Screenshot 2022-10-20 at 13 35 47" src="https://user-images.githubusercontent.com/275961/196950088-b84faf96-bb13-422c-96b6-c15d75360109.png">
